eng/pipelines/docs.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           npm install yargs
         displayName: "Install yargs"
 
-      - powershell: |
+      - pwsh: |
```

Comment:
I'm a huge fan of leveraging Powershell6, but I recently got bitten super hard by `PSCore` vs `PS5.1` issues with the git release script. `MaximumRetryCount ` is just like the one that recently got me, which was that I was using a function `FollowRelLink` in `Invoke-Webrequest`.

Funnily enough, that's exactly why you swapped this to PSCore :P  

If we're ok with the inconsistency (which definitely saves needing to write some dumb boilerplate over simply using a newer version), then I'm good with this. Seems logical to me.

Reply:
What do you mean by "the inconsistency"?

Reply:
We need to cut over all `pshell` tasks to `pwsh`. Until that's done I'd call our CI across builds inconsistent, because you'll need to remember which version you're executing (and make certain you get the proper one!)

Reply:
I agree we should use `pwsh` consistently everywhere.  I'm planning to update the 4 main repos today.

Reply:
We probably need to update the release pipelines at some point as well.